tox.ini

```diff
@@ -1,13 +1,18 @@
 [tox]
 isolated_build = true
-envlist = py37-django32,{38,39,310}-django{22,32,40,41},report
+envlist =
+    py37-django32
+    py{38,39,310}-django{32,40,41,42}
+    py311-django{41,42}
+    report
```

Comment:
the env declaration needed some fixing - 7e293d9a139a4b712b98835faacf0251722dc303 can be cherry-picked